api/clients/accountant.go

```diff
@@ -36,16 +37,15 @@ type accountant struct {
 	usageLock         sync.Mutex
 	cumulativePayment *big.Int
 
-	paymentSigner core.PaymentSigner
-	numBins       uint32
+	numBins uint32
 }
 
 type BinRecord struct {
 	Index uint32
 	Usage uint64
 }
 
-func NewAccountant(reservation *core.ActiveReservation, onDemand *core.OnDemandPayment, reservationWindow uint32, pricePerSymbol uint32, minNumSymbols uint32, paymentSigner core.PaymentSigner, numBins uint32) *accountant {
+func NewAccountant(accountID string, reservation *core.ActiveReservation, onDemand *core.OnDemandPayment, reservationWindow uint32, pricePerSymbol uint32, minNumSymbols uint32, numBins uint32) *accountant {
```

Comment:
Are `reservationWindow`, `pricePerSymbol`, `minNumSymbols`, `numBins` different for each account? 
Aren't these globally available somewhere?

Reply:
`reservationWindow`, `pricePerSymbol`, `minNumSymbols` should be set through the contract. I linked a PR in the previous comment about this. `numBins` can be set by the accountants preferences, doesn't really matter but restricted by a minimum of 3

Reply:
This is just a nice functionality considered for disperser clients who want to track more history in-memory. The data isn't really used and not critical